restcomm/restcomm.application/src/main/java/org/restcomm/connect/application/Bootstrapper.java

```diff
@@ -246,48 +244,66 @@ public UntypedActor create() throws Exception {
 
     }
 
+    private ActorRef sdrService(final Configuration configuration, final ClassLoader loader) throws Exception {
+        final String className = configuration.getString("sdr-service[@class]");
+        if (className != null) {
+            final Props props = new Props(new UntypedActorFactory() {
+                @Override
+                public Actor create() throws Exception {
+                    return (Actor) new ObjectFactory(loader).getObjectInstance(className);
+                }
+            });
+            ActorRef sdr = system.actorOf(props);
+            sdr.tell(new StartSdrService(configuration), null);
```

Comment:
do we need some explicit cleanup of this service on application closing? do we need to send some message or close the actor on application closing?

Reply:
Not here for sure. akka actors have their own close context mechanizm.